contract/src/msgs/data_requests/sudo/remove_requests.rs

```diff
@@ -26,7 +26,7 @@ fn amount_to_tokens(amount: Uint128, token: &str) -> Coin {
 
 fn remove_request(
     dr_id_str: String,
```

Comment:
Does it make sense to update the funciton name to something that describes that distribution messages are converted to bank messages?

Reply:
I think that this kind of function could have some function documentation about what is does.

Reply:
`remove_request_and_process_distributions`? It's long :x but maybe that's fine? Or if someone better at naming has an idea please help